pincer/objects/guild/guild.py

```diff
@@ -1236,6 +1237,165 @@ async def modify_user_voice_state(
             }
         )
 
+    async def get_audit_log(self) -> AuditLog:
+        """|coro|
+        Returns an audit log object for the guild.
+        Requires the ``VIEW_AUDIT_LOG`` permission.
+
+        Returns
+        -------
+        :class:`~pincer.objects.guild.audit_log.AuditLog`
+            The audit log object for the guild.
+        """
+        return AuditLog.from_dict(
+            construct_client_dict(
+                self._client,
+                await self._http.get(f"guilds/{self.id}/audit-logs")
+            )
+        )
+
+    async def get_emojis(self) -> AsyncGenerator[Emoji, None]:
+        """|coro|
+        Returns an async generator of tl emojis in the guild.
+
+        Yields
+        ------
+        :class:`~pincer.objects.guild.emoji.Emoji`
+            The emoji object.
+        """
+        data = await self._http.get(f"guilds/{self.id}/emojis")
+        for emoji_data in data:
+            yield Emoji.from_dict(
+                construct_client_dict(self._client, emoji_data)
+            )
+
+    async def get_emoji(self, id: Snowflake) -> Emoji:
+        """|coro|
+        Returns an emoji object for the given ID.
+
+        Parameters
+        ----------
+        id : :class:`~pincer.utils.snowflake.Snowflake`
+            The ID of the emoji
+
+        Returns
+        -------
+        :class:`~pincer.objects.guild.emoji.Emoji`
+            The emoji object.
+        """
+        return Emoji.from_dict(
+            construct_client_dict(
+                self._client,
+                await self._http.get(f"guilds/{self.id}/emojis/{id}")
+            )
+        )
+
+    async def create_emoji(
+        self,
+        *,
+        name: str,
+        image: str,
+        roles: List[Snowflake],
+        reason: Optional[str] = None
+    ) -> Emoji:
+        """|coro|
+        Creates a new emoji for the guild.
+        Requires the ``MANAGE_EMOJIS_AND_STICKERS`` permission.
+
+        Emojis and animated emojis have a maximum file size of 256kb.
+        Attempting to upload an emoji larger than this limit will fail.
```

Comment:
Maybe add some error handling?

Reply:
Not quite sure what I could do really; the docs say that doing so will
> *return 400 Bad Request and an error message, but not a JSON status code*

Reply:
Guess you can leave it

Reply:
Can I resolve this then?

Reply:
Yeah, we can add pillow support later tbh. Just mark it as Partially in the spreadsheet.